.yarnrc.yml

```diff
@@ -1,5 +1,7 @@
-plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
-    spec: "@yarnpkg/plugin-interactive-tools"
+compressionLevel: mixed
 
-yarnPath: .yarn/releases/yarn-3.6.1.cjs
+enableGlobalCache: false
```

Comment:
Just curious, is there any reason for this?

Reply:
It is automatically generated by Yarn 4 when migrating from Yarn 3 config. The default is now `true` in Yarn 4, but it was `false` in Yarn 3.